### PR TITLE
Rebase off Mesos 1.0.x branch

### PR DESCRIPTION
**Pull request by adam-mesos**
Targeting for 1.8.5

### CODE REVIEW

File: packages/mesos/README.md [hunk 1]
```diff
@@ -10,3 +10,6 @@
 <li>[3dd498d797a44593d376b4d50a33e2b8214c0dd4] Split src/openssl.hpp adding include/process/ssl/flags.hpp. (landing in 1.1.0)
 <li>[f6581f2e8fcb07074fca9e72b3134476f6fc74ab] Updated includes to follow the SSL flag split. (landing in 1.1.0)
 <li>[a22b5dd552168a8dd925682f3b349059e10fc3d3] Modified network file setup in `network/cni` isolator. (landing in 1.1.0)
+<li>[3e703af02bc309a9d6653392d02627bf3d3ce930] Updated logrotation module to use `os::pagesize()`. (landing in 1.1.0)
+<li>[1d45ba9f4f7f943d24282e0a91fb01c68815d6c3] Fixed sign comparisons in logrotate module. (landing in 1.1.0)
+<li>[b546e7544c04a6d3e0da87cd6207ae8f4d827add] Added a way to set logrotate settings per executor. (landing in 1.1.0)
```
Comment (adam-mesos) [inline packages/mesos/README.md]: These were already on dcos-mesos-1.0.1 for @kaysoky 's soon-to-be merged #572 and I didn't want to lose the commits/readme by not including them here just because they're not in dcos/dcos master yet. If #572 lands first, this will rebase into nothingness

File: packages/mesos/buildinfo.json [hunk 2]
```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "a22b5dd552168a8dd925682f3b349059e10fc3d3",
-    "ref_origin" : "dcos-mesos-1.0.1"
+    "ref": "0950e8664d1b181e82dabe958d9cc1d43a5d7612",
+    "ref_origin" : "dcos-mesos-1.0.x-7b322c9"
```
Comment (adam-mesos) [inline packages/mesos/buildinfo.json]: I just rebased off the HEAD of the 1.0.x branch, but it would be lovely if we had an official 1.0.2 (rc at least) to use for 1.8.5.

Comment (adam-mesos) [inline packages/mesos/buildinfo.json]: The chance of an official 1.0.2 is unlikely, so I still like this strategy of using the WIP 1.0.x branch at this SHA, which will eventually become 1.0.2 (after a few more unrelated fixes land), rather than cherry-pick just the 2 we need for executor SSL environment on top of 1.0.1.
TL;DR: We should use this branch and SHA in 1.8.5.

File: packages/mesos/buildinfo.json [hunk 3]
```diff
@@ -3,8 +3,8 @@
   "single_source" : {
     "kind": "git",
     "git": "https://github.com/mesosphere/mesos",
-    "ref": "b546e7544c04a6d3e0da87cd6207ae8f4d827add",
-    "ref_origin" : "dcos-mesos-1.0.1"
+    "ref": "103689ce3deba22b46de0f7161f0f8b85b12a894",
+    "ref_origin" : "dcos-mesos-1.0.x-c885b26"
```
Comment (adam-mesos) [inline packages/mesos/buildinfo.json]: Bumped again, to the latest 1.0.x branch SHA, to include:
- @kaysoky 's logrotate changes from `dcos-mesos-1.0.1`
- Containerizer env fixes for MESOS-6075 / MESOS-6235
- Recent MESOS-6245 fix for "driver based schedulers to ACK updates from HTTP executors"

File: packages/mesos/README.md (hunk 1 above)
Comment (adam-mesos) [inline packages/mesos/README.md]: #572 landed, so these lines are no longer part of this diff (already in master).

File: packages/mesos/buildinfo.json (hunk 2 above)
Comment (adam-mesos) [inline packages/mesos/buildinfo.json]: Bumped the SHA to the latest 1.0.x, so we'll stick with that branch until a 1.0.2 rc comes out.

**Comment by adam-mesos** [comment]
@cmaloney Rebased after @kaysoky 's #572 landed, looking green.
